molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+  requirements_file: tests/requirements.yml
+
+driver:
+  name: docker
+
+vagrant:
+  platforms:
+    - name: centos7
+      box: centos/7
+  providers:
+    - name: virtualbox
+      type: virtualbox
+      options:
+        memory: 512
+        cpus: 1
+  instances:
+    - name: jekyll-build
+
+docker:
+  containers:
+  - name: jekyll-build
+    image: centos/systemd
+    image_version: latest
+    privileged: True
```

Comment:
Yes, thanks for spotting the copy-and-paste of another roles (depending on `systemd` most likely) - pushed as 204ba1b.